examples/nextjs/pages/second.tsx

```diff
@@ -0,0 +1,33 @@
+import { NextPageContext } from 'next'
+import React from 'react'
+
+import { getNhostSession, NhostSession } from '@nhost/nextjs'
+import { useAccessToken, useAuthenticated, useUserData } from '@nhost/react'
+
+import { BACKEND_URL } from '../helpers'
+
+export async function getServerSideProps(context: NextPageContext) {
+  const nhostSession = await getNhostSession(BACKEND_URL, context)
+  return {
+    props: {
+      nhostSession
+    }
+  }
+}
+
+const SecondPage: React.FC<{ initial: NhostSession }> = () => {
+  const isAuthenticated = useAuthenticated()
+  const user = useUserData()
+  const jwt = useAccessToken()
```

Comment:
I think we should always use "access token" instead of "jwt" for consistency. This has to be changed in more places than here.

Reply:
ok